gc/base/SweepHeapSectioning.hpp

```diff
@@ -41,8 +41,7 @@ class MM_HeapRegionDescriptor;
 class MM_SweepHeapSectioning : public MM_BaseVirtual {
 private:
 protected:
-	MM_ParallelSweepChunkArray* _head; /**< head pointer to a list of Chunk arrays */
-	uintptr_t _totalUsed; /**< total elements used across all arrays */
+	MM_ParallelSweepChunkArray* _tail; /**< tail pointer to a list of chunk arrays */
 	uintptr_t _totalSize; /**< total elements available across all arrays */
 
 	MM_ParallelSweepChunkArray* _baseArray; /**< pointer to the base array allocated at initialization */
```

Comment:
you can also append a comment, that his serves as a head of the list

Reply:
Should we just change the `_baseArray` to `_head`?

Reply:
no strong preference from my side - I'll leave it up to you

Reply:
Changed to `_head`